translations/ui-checkout/en.json

```diff
@@ -25,6 +25,7 @@
   "title": "Title",
   "loanPolicy": "Loan policy",
   "due.date": "Due date",
+  "due.date.change": "System Calculated due date adjustment",
```

Comment:
Should be "System-calculated due-date adjustment" I think (when two nouns modify a third, the first two are hyphenated). At the very least, it should be "System calculated...".